docs/development.md

```diff
@@ -0,0 +1,44 @@
+# Wordy Developer Docs
+
+## Status
+
+[![Build Status](https://travis-ci.org/ustwo/wordy-slack-bot.svg?branch=master)](https://travis-ci.org/ustwo/wordy-slack-bot)
+
+## Configuration
+
+ * `SLACK_TOKEN`: Slack Bot token. You can create a bot and get a token in https://YOURSLACK.slack.com/services/new/bot. For more information you can read [Slack Bots documentation](https://api.slack.com/bot-users).
```

Comment:
the URL `https://YOURSLACK.slack.com/services/new/bot` should be wrapped in backticks (`) so Github does not autolink it.